src/streamed_orderbook.ts

```diff
@@ -0,0 +1,267 @@
+import assert from 'assert'
+import Web3 from 'web3'
+import { EventData } from 'web3-eth-contract'
+import { abi } from '@gnosis.pm/dex-contracts/build/contracts/BatchExchange.json'
+import { BatchExchange } from '@gnosis.pm/dex-contracts/build/types/BatchExchange'
+
+const STARTING_BLOCK = 9340147
```

Comment:
I assume this is the block we deployed the exchange contract? Is there any way to get that programatically?

Reply:
Yes, I was just lazy to write it up. It's in the `networks` section of the truffle artifact.